File: PEMROGRAMAN 1/PRAKTIKUM MODUL 2/Bahasa Python/PRAK203-2410817310009-Muhammad_Ghazi_Rakhmadi.py
# test case 1
a, b, i, j, x, y = map(float, input("a, b, i, j, x, dan y :\n").split())
hasil = (a - b) * (i / j) - (x + y)
print(f"Hasil : {hasil:.3f}\n")
# test case 2
a1, b1 = map(float, input("a, b, i, j, x, dan y :\n").split())
i1, j1 = map(float, input().split())
x1, y1 = map(float, input().split())
hasil1 = (a1 - b1) * (i1 / j1) - (x1 + y1)
print(f"Hasil : {hasil1:.3f}")

File: PEMROGRAMAN 1/PRAKTIKUM MODUL 2/Bahasa C/PRAK203-2410817310009-Muhammad_Ghazi_Rakhmadi.c
#include <stdio.h>
int main () {
 //test case 1
 float a1, b1, i1, j1, x1, y1;
 printf("a, b, i, j, x, dan y :\n");
 scanf("%f", &a1);
 scanf("%f", &b1);
 scanf("%f", &i1);
 scanf("%f", &j1);
 scanf("%f", &x1);
 scanf("%f", &y1);
 printf("Hasil : %.3f\n", (a1 - b1) * (i1 / j1) - (x1 + y1));
 printf("\n");
 //test case 2
 float a2, b2, i2, j2, x2, y2;
 printf("a, b, i, j, x, dan y :\n");
 scanf("%f %f", &a2, &b2);
 scanf("%f %f", &i2, &j2);
 scanf("%f %f", &x2, &y2);

 printf("Hasil : %.3f", (a2 - b2) * (i2 / j2) - (x2 + y2));
 return 0;
}